iframe-demo.html

```diff
@@ -31,5 +31,10 @@ <h2><code>/iframe/assay-barchart</code></h2>
       <iframe src="http://localhost:5001/iframe/assay-barchart" height="450" width="800" scrolling="no"></iframe>
       <!-- The element has more whitespace at the bottom than you probably need.-->
     </div>
+
+    <h2><code>/iframe/organ</code></h2>
+    <div>
+      <iframe src="http://localhost:5001/iframe/organ?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0002113" height="604" width="916" scrolling="no"></iframe>
```

Comment:
The intent is for this to just work locally?

Reply:
Nope! Just sloppy.

Reply:
oh, sorry: Yes, this is correct as-is. Maybe there's a better approach, but with all these these iframes, I just want to be able to sanity check them. For the other ones, it's also intended as a resource for Bill, so he can see which dimensions work well.